code/modules/food/recipes_microwave.dm

```diff
@@ -386,6 +386,18 @@
 		)
 	result = /obj/item/weapon/reagent_containers/food/snacks/rofflewaffles
 
+/datum/recipe/rofflewaffles
+	reagents = list(PSILOCYBIN = 5, FLOUR = 10)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/egg,
+		/obj/item/weapon/reagent_containers/food/snacks/egg,
+		)
+	result = /obj/item/weapon/reagent_containers/food/snacks/rofflewaffles
```

Comment:
@DeityLink It seems you duplicated the rofflewaffles recipe for some reason.

Reply:
woops